examples/onft721-zksync/contracts/mocks/MyONFT721Mock.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.22;
+
+import { MyONFT721 } from "../MyONFT721.sol";
+
+// @dev WARNING: This is for testing purposes only
+contract MyONFT721Mock is MyONFT721 {
+    constructor(
+        string memory _name,
+        string memory _symbol,
+        address _lzEndpoint,
+        address _delegate
+    ) MyONFT721(_name, _symbol, _lzEndpoint, _delegate) {}
+
+    function mint(address _to, uint256 _amount) public {
```

Comment:
This was my mistake in the original mock, can you please change `_amount` to `_id`.  I have changed in the original one too.